src/android/StatusBar.java

```diff
@@ -51,26 +56,57 @@ public void initialize(final CordovaInterface cordova, CordovaWebView webView) {
         LOG.v(TAG, "StatusBar: initialization");
         super.initialize(cordova, webView);
 
-        this.cordova.getActivity().runOnUiThread(new Runnable() {
+        Activity activity = cordova.getActivity();
+        ActivityAssistant.getInstance().assistActivity(activity);
+
+        activity.runOnUiThread(new Runnable() {
             @Override
             public void run() {
+                doOverlay = preferences.getBoolean("StatusBarOverlaysWebView", false);
+
                 // Clear flag FLAG_FORCE_NOT_FULLSCREEN which is set initially
                 // by the Cordova.
                 Window window = cordova.getActivity().getWindow();
                 window.clearFlags(WindowManager.LayoutParams.FLAG_FORCE_NOT_FULLSCREEN);
 
-                // Read 'StatusBarOverlaysWebView' from config.xml, default is true.
-                setStatusBarTransparent(preferences.getBoolean("StatusBarOverlaysWebView", true));
+                // Allows app to overlap cutout area from device when in landscape mode (same as iOS)
+                // More info: https://developer.android.com/reference/android/R.attr.html#windowLayoutInDisplayCutoutMode
+                if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+                    window.getAttributes().layoutInDisplayCutoutMode = WindowManager.LayoutParams.LAYOUT_IN_DISPLAY_CUTOUT_MODE_SHORT_EDGES;
+                }
+
+                // Allows app to overlap cutout area from device when in landscape mode (same as iOS)
+                // More info: https://developer.android.com/reference/android/R.attr.html#windowLayoutInDisplayCutoutMode
+                if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+                    window.getAttributes().layoutInDisplayCutoutMode = WindowManager.LayoutParams.LAYOUT_IN_DISPLAY_CUTOUT_MODE_SHORT_EDGES;
+                }
+                                // Added to override logic if plugin is installed in OutSystems Now app.
```

Comment:
This logic can be simplified given that `OutSystemsNow` no longer exists (RIP) and the minimum version supported is higher than all versions specified here.

Reply:
Although I believe this is not the scope of the issue, we could have backlog to tackle this so we don't forget about it. It would also decrease the differences to the upstream.